pegasus/sites.v3/codeprojects.org/views/404.haml

```diff
@@ -0,0 +1,20 @@
+---
+title: Page Not Found
+nav: blank
+rightbar: blank
+---
+%h1 The requested page was not found
+%p{style: 'font-size: 1.5em; line-height: 1.5em;'} Check the web address and resubmit or choose a link from the list below.
+%ul{style: 'font-size: 1.25em; line-height: 1.5em; min-height: 240px;'}
+  %li<
+    %a{href: '//code.org/learn'} Tutorials for students
+  %li<
+    %a{href: '//code.org/teach'} Educator resources
+  %li<
+    %a{href: '//code.org/promote'} Ways to promote computer science education
+  %li<
+    %a{href: '//code.org/help'} How to help
+  %li<
+    %a{href: '//code.org/about'} Information about Code.org and its mission
+  %li<
+    %a{href: 'http://codeorg.tumblr.com/'} Code.org’s blog
```

Comment:
`codeorg.tumblr.com` just redirects to `blog.code.org` - maybe should use `//blog.code.org` here?